serverless.yml

```diff
@@ -390,6 +390,7 @@ functions:
 
   node_api_get_version:
     handler: handlers/node_api.get_version
+    timeout: 6
```

Comment:
Maybe add a comment indicating this is informed in seconds?

Reply:
I will let it unchanged, because if I add this comment here, I would have to add this comment in all other place.